engine/src/main/java/org/destinationsol/game/SaveManager.java

```diff
@@ -15,30 +15,57 @@
  */
 package org.destinationsol.game;
 
+import java.io.FileNotFoundException;
+import java.io.PrintWriter;
+import java.io.UnsupportedEncodingException;
 import java.nio.file.Paths;
 import java.util.ArrayList;
+import java.util.List;
 
 import org.destinationsol.IniReader;
 import org.destinationsol.files.HullConfigManager;
 import org.destinationsol.game.item.Gun;
+import org.destinationsol.game.item.ItemContainer;
 import org.destinationsol.game.item.ItemManager;
 import org.destinationsol.game.item.MercItem;
 import org.destinationsol.game.item.SolItem;
+import org.destinationsol.game.ship.SolShip;
 import org.destinationsol.game.ship.hulls.HullConfig;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 import com.badlogic.gdx.files.FileHandle;
+import com.badlogic.gdx.math.Vector2;
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import com.google.gson.JsonObject;
 
 public class SaveManager {
     private static final String FILE_NAME = "prevShip.ini";
 
-    public static void writeShip(HullConfig hull, float money, ArrayList<SolItem> items, SolGame game) {
+    private static Logger logger = LoggerFactory.getLogger(SaveManager.class);
+
+    public static void writeShips(HullConfig hull, float money, ArrayList<SolItem> itemsList, SolGame game) {
         String hullName = game.getHullConfigs().getName(hull);
+
+        writeMercs(game);
+
+        String items = itemsToString(itemsList);
+        
+        Vector2 pos = game.getHero().getPosition();
+        
+        IniReader.write(FILE_NAME, "hull", hullName, "money", (int) money, "items", items, "x", pos.x, "y", pos.y);
+    }
+
+    /**
+     * Converts  list of SolItems to a string of items to be saved.
```

Comment:
One space too many.

Also,
`Encodes the given list of SolItems as a string.`

(Throw in a "suitable for saving" at the end if you really want to)